release_notes/ose_3_1_release_notes.adoc

```diff
@@ -549,14 +549,16 @@ properly cached and used for volume management.
 ==== Known Issues
 
 https://bugzilla.redhat.com/show_bug.cgi?id=1293578[*BZ#1293578*]::
-There was an issue with OpenShift Enterprise 3.1.1 which involved hosts where their 
-hostnames resolves to an IP address that's not local to the host would run into
-problems with liveness and readiness probes on newly created HAProxy Routers. This was resolved in
+There was an issue with OpenShift Enterprise 3.1.1, where hosts with hostnames 
+that resolved to IP addresses that were not local to the host would run into
+problems with liveness and readiness probes on newly-created HAProxy Routers. 
+This was resolved in
 https://access.redhat.com/errata/product/290/ver=3.1/rhel---7/x86_64/RHBA-2016:0293[RHBA-2016:0293]
-by configuring the probes to use 'localhost' as the hostname for pods utilizing hostPorts.
+by configuring the probes to use 'localhost' as the hostname for pods utilizing 
+hostPorts.
 
-If you created a router under the affect version you should add 'host: localhost'
-if your liveness or readiness probes unexpectedly failing for your router.
+If you created a router under the affected version, and your liveness or 
+readiness probes unexpectedly fail for your router, then add 'host: localhost': 
```

Comment:
See comments in `manual_upgrades.adoc`,  as they're all the same for this topic too.